cocos/core/geometry/sphere.ts

```diff
@@ -63,8 +61,8 @@ export class Sphere {
      * clone a new sphere
      * @zh
      * 克隆一个新的 sphere 实例。
-     * @param {Sphere} p 克隆的目标。
-     * @return {Sphere} 克隆出的示例。
+     * @param {Sphere} p @en Sphere which cloned from. @zh 克隆的目标。
+     * @return {Sphere} @en Sphere which cloned to. @zh 克隆出的示例。
```

Comment:
No need to indicate type of the parameter

`@param p @en ... @zh ...`
`@return ...`

Please remove all `{type}` in the file